templates/titian-mosaic.template

```diff
@@ -29,7 +29,7 @@ Metadata:
       - pEC2KeyPair
       - pAppServerInstanceType
     - Label:
-        default: Amazon RDS configuration
+        default: Amazon RDS Configuration
```

Comment:
Can you please revert the capitalization of every word? It doesn't follow AWS writing guidelines.

Reply:
When I look at the main branch/last commit, these appear to be capitalized in commit [b906502].
I will make my way back through these and revert for this pull.